app/page.tsx

```diff
@@ -49,21 +44,175 @@ export default function IndexPage() {
 
           <div className="flex flex-col items-center gap-4 mt-10">
             <p className="max-w-[700px] text-lg text-slate-900 dark:text-white sm:text-xl">
-              Our community is based on open-source principles and e're focused on shaping the future of the web. Join us as we collaborate on projects, share knowledge, and build community.
+              Our community is based on open-source principles and we're focused on shaping the
+              future of the web. Join us as we collaborate on projects, share knowledge, and build
+              community.
             </p>
           </div>
         </div>
 
         <div className="absolute">
-          <Image src="/12.png" width={1200} height={1200} alt="logo" className="mix-blend-difference" />
+          <Image
+            src="/12.png"
+            width={1200}
+            height={1200}
+            alt="logo"
+            className="mix-blend-difference"
+          />
         </div>
 
-        <Image src='/flying-card-001.png' alt="flying card 001" width={135} height={165} className="absolute top-[12rem] left-[12rem]" />
+        <Image
+          src="/flying-card-001.png"
+          alt="flying card 001"
+          width={135}
+          height={165}
+          className="absolute top-[12rem] left-[12rem]"
+        />
 
-        <Image src='/flying-card-002.png' alt="flying card 001" width={208} height={259} className="absolute bottom-[15rem] right-[12rem]" />
+        <Image
+          src="/flying-card-002.png"
+          alt="flying card 001"
+          width={208}
+          height={259}
+          className="absolute bottom-[15rem] right-[12rem]"
+        />
         <CursorHoverster />
       </section>
 
+      {/* Headline */}
+      <section className="container grid items-center gap-6 pb-8 pt-6 md:py-10 justify-center text-center mt-20">
+        <div className="flex max-w-[980px] flex-col items-center gap-2">
+          <h3 className="text-xl font-extrabold leading-tight tracking-tighter sm:text-2xl md:text-3xl lg:text-2xl px-32">
+            Become an integral part of a dynamic and vibrant network of like-minded developers
+          </h3>
+          <div className="flex gap-4 gap-x-4 mt-4">
+            <div className="flex items-center">
+              <Icons.twitter className="h-5 w-5 mr-1 fill-current cursor-pointer" />
+              <span className="cursor-pointer">Twitter</span>
+            </div>
+
+            <div className="flex items-center">
+              <Icons.discord className="h-5 w-5 mr-1 fill-current cursor-pointer" />
+              <span className="cursor-pointer">Discord</span>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      {/* Blurbs */}
+      <section className="flex py-0 px-40">
```

Comment:
```suggestion
      <section className="flex justify-center py-0 px-40">
```

You can add `justify-center` to center the section. :)

Reply:
Thanks for the suggestion. I have added the suggested changes.